third_party/terraform/website/docs/r/dns_record_set.markdown

```diff
@@ -117,6 +117,23 @@ resource "google_dns_managed_zone" "prod" {
 }
 ```
 
+### Adding a CNAME record
+ The list of `rrdatas` should only contain a single string corresponding to the Canonical Name intended.
+ ```hcl
+resource "google_dns_record_set" "cname" {
+  name = "frontend.${google_dns_managed_zone.prod.dns_name}"
+  managed_zone = "${google_dns_managed_zone.prod.name}"
+  type = "CNAME"
+  ttl  = 300
+  rrdatas = ["frontend.mydomain.com."]
```

Comment:
you're going to end up with extra generation stuff for unrelated PRs because the one you merged has a newline before this :-/

Reply:
Ah oh no! I tried to fix it using the MM downstream but Git outsmarted me. Since it's just whitespace we'll just merge the next Terraform MM PR with those changes included.